src/stanza-definitions/xep0051.ts

```diff
@@ -0,0 +1,45 @@
+import { Stanzas } from 'stanza';
+import { childText, DefinitionOptions } from 'stanza/jxt';
+
+// XEP-0051: Connection Transfer
+// Source: https://xmpp.org/extensions/xep-0051.html
```

Comment:
Should we instead point this to RFC 6120 since it now supersedes XEP-0051?

Reply:
As followup to this, the relevant section is 4.9.3.19: https://www.rfc-editor.org/rfc/rfc6120#section-4.9.3.19. The details for that are different (e.g. it’s a stream error of see-other-host), and Hawk still explicitly uses XEP-0051 at this time.